Compute/PollingAgent/src/Program.cs

```diff
@@ -48,6 +48,7 @@
 
 using Serilog;
 
+using Pollster = ArmoniK.Core.Common.Injection.Options.Pollster;
```

Comment:
Not sure that options should take precedence over the concrete types they apply to.

Reply:
I can't. Resharper is configured like so. The alternative is more verbose so I think r# chooses this one